boards/x86/tinytile/tinytile.dts

```diff
@@ -20,11 +20,7 @@
 	chosen {
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
-#ifdef CONFIG_USB_UART_CONSOLE
-		zephyr,console = &usb_cdc;
-#else
 		zephyr,console = &uart1;
-#endif
 		zephyr,bt-uart = &uart0;
```

Comment:
Does the board has only USB Serial console only? How the board works after that?

Reply:
> Does the board has only USB Serial console only?

no, it does have UART.

Board will work exactly the same way, if you want to use CDC, then you will need an overlay.